backwards-compat-tests/scripts/run_local.sh

```diff
@@ -0,0 +1,75 @@
+#!/usr/bin/env bash
+
+set -euo pipefail
+
+ORIGINAL_BRANCH=$(git branch --show-current)
+
+on_exit() {
+  # set +e
+  git checkout "$ORIGINAL_BRANCH"
+  # ./risedev k
+  # set -e
+}
+
+trap on_exit EXIT
+
+OLD_TAG=1.0.0
+NEW_TAG=1.1.0
```

Comment:
This adds a new task for the release process. 🥵 Consider making it an argument/variable ?